fe/fe-core/src/main/java/com/starrocks/qe/ShowExecutor.java

```diff
@@ -1545,9 +1545,12 @@ private void handleShowAnalyze() {
         resultSet = new ShowResultSet(stmt.getMetaData(), rows);
     }
 
-    private void handleShowWorkGroup() {
+    private void handleShowWorkGroup() throws AnalysisException {
         ShowWorkGroupStmt showWorkGroupStmt = (ShowWorkGroupStmt) stmt;
         List<List<String>> rows = GlobalStateMgr.getCurrentState().getWorkGroupMgr().showWorkGroup(showWorkGroupStmt);
+        if(showWorkGroupStmt.getName() != null && rows.isEmpty()){
```

Comment:
It will be better to throw this exception in `showWorkGroup(showWorkGroupStmt);` function when the specified database is not found.

Reply:
yeah, you are right.